docs/data/tree-view/simple-tree-view/selection/CheckboxSelection.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import { SimpleTreeView } from '@mui/x-tree-view/SimpleTreeView';
+import { TreeItem } from '@mui/x-tree-view/TreeItem';
+
+export default function CheckboxSelection() {
+  return (
+    <Box sx={{ height: 264, flexGrow: 1, maxWidth: 400 }}>
```

Comment:
```suggestion
    <Box sx={{ minHeight: 264, flexGrow: 1, maxWidth: 400 }}>
```

<img width="675" alt="SCR-20240523-qrny" src="https://github.com/mui/mui-x/assets/3165635/4ec85e0e-100f-4a6c-b75f-767f7a28ef23">

Also true with `CheckboxMultiSelection.tsx`

Reply:
#13237